spanner/api/Spanner.Samples/UpdateUsingDmlReturningAsync.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_update_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class UpdateUsingDmlReturningAsyncSample//TODO update sample tag
+{
+    public async Task<int> UpdateUsingDmlReturningAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        //Updates records from Venues table satisfying a particular condition and returns the VenueId
```

Comment:
nit: Same here and in other samples as well. I am not commenting at every place. Please take a look and correct.